index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+    <title>Fred's Personal Portfolio</title>
+</head>
+<body>
+    <div class="header-container">
+        <header class="toolbar-section">
+            <nav class="nav-menu">
+                <a href="" class="logo"><img src="images/My-Logo.svg" alt=""></a>
+                <a href="" class="menu-icon"><img src="images/Icon-Menu.svg" alt=""></a>
+            </nav>
+        </header>
+        <section class="headline-section">
```

Comment:
- [ ]  [OPTIONAL] right now your design work great with 375 px screens in width however it will be broken on some other mobile screens so please try to make it responsive on all mobile screens